models/repo.go

```diff
@@ -530,12 +530,7 @@ func (repo *Repository) DeleteWiki() error {
 }
 
 func (repo *Repository) deleteWiki(e Engine) error {
```

Comment:
this will make func repo.DeleteWiki() not work properly (L528)

Reply:
I propose to delete this func here copy/paste sql code into `DeleteRepository` and the current function into `DeleteWiki`

Reply:
UPDATE: we shuld find func who use repo.DeleteWiki() and switch them to wiki_service.DeleteWiki(repo)

Reply:
I have replaced all places.

Reply:
@lunny commited my changes :)